src/main/java/com/knu/buga1chuk/data_structure/FlexibleArrayDemo.java

```diff
@@ -24,11 +29,11 @@ public static void main(String[] args) {
         flexArray.print();
 
         int removedElement = flexArray.subtractElement();
-        System.out.println(removedElement);
+        LOG.info(String.valueOf(removedElement));
```

Comment:
Extract the string value into a separate variable.
And add some message.

Reply:
fix